app/app.go

```diff
@@ -517,6 +517,7 @@ func NewAxelarApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 		ante.NewValidateValidatorDeregisteredTssDecorator(tssK, nexusK, snapK),
 		ante.NewCheckRefundFeeDecorator(app.interfaceRegistry, accountK, stakingK, snapK, axelarnetK),
 		ante.NewCheckProxy(snapK),
+		ante.NewRestrictedTx(tssK),
```

Comment:
why are we using the ante handler on the tss keeper to veto updates to the governance keys, given that the tss module is part of axelar-core?